<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-22T17:40:21 IST" name="Default suite" finished-at="2023-05-22T17:42:58 IST" duration-ms="156157">
    <groups>
    </groups>
    <test started-at="2023-05-22T17:40:21 IST" name="Default test" finished-at="2023-05-22T17:42:58 IST" duration-ms="156157">
      <class name="testcase.HomePageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:40:22 IST" name="setup" finished-at="2023-05-22T17:41:13 IST" duration-ms="51538" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ValidateDesktopModule()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:41:13 IST" name="ValidateDesktopModule" finished-at="2023-05-22T17:41:13 IST" duration-ms="105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateDesktopModule -->
        <test-method is-config="true" signature="close()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:41:13 IST" name="close" finished-at="2023-05-22T17:41:13 IST" duration-ms="306" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:41:13 IST" name="setup" finished-at="2023-05-22T17:42:04 IST" duration-ms="50488" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ValidatePageTitle()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:42:04 IST" name="ValidatePageTitle" finished-at="2023-05-22T17:42:04 IST" duration-ms="12" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Yours Store] but found [Your Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Yours Store] but found [Your Store]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcase.HomePageTest.ValidatePageTitle(HomePageTest.java:27)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidatePageTitle -->
        <test-method is-config="true" signature="close()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:42:04 IST" name="close" finished-at="2023-05-22T17:42:04 IST" duration-ms="83" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:42:04 IST" name="setup" finished-at="2023-05-22T17:42:57 IST" duration-ms="53168" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ValidateTitleDisplay()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:42:57 IST" name="ValidateTitleDisplay" finished-at="2023-05-22T17:42:57 IST" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateTitleDisplay -->
        <test-method is-config="true" signature="close()[pri:0, instance:testcase.HomePageTest@c267ef4]" started-at="2023-05-22T17:42:57 IST" name="close" finished-at="2023-05-22T17:42:58 IST" duration-ms="358" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- testcase.HomePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
